starknet_py/net/schemas/rpc.py

```diff
@@ -83,6 +76,8 @@ def make_dataclass(self, data, **kwargs):
 
 
 class L1toL2MessageSchema(Schema):
+    nonce = Felt(data_key="nonce", required=True)
+    selector = Felt(data_key="selector", required=True)
```

Comment:
Is it okay for these to be marked as required?
Won't it break v0.3?

Reply:
This schema could be as well removed, because the field connected to it isn't returned in FullNode, but is in Gateway